app/kb.py

```diff
@@ -58,11 +57,8 @@ def _load_foods(self):
                     new_food.nutritional_data = nutritional_data[new_food.food_id]
                 self.foods.append(new_food)
 
-    def _load_preparation_terms(self):
-        self.preparation_terms = set(read_txt_lines_into_list('kb_data/preparation_terms.txt'))
-
-    def _load_preparation_descriptors(self):
-        self.preparation_descriptors = set(read_txt_lines_into_list('kb_data/preparation_descriptors.txt'))
+    def _load_cooking_terms(self):
+        self.cooking_terms = set(read_txt_lines_into_list('kb_data/cooking_terms.txt'))
```

Comment:
Right, so you also need to look at the __init__ method and change the global fields, k? k.